### PR TITLE
changing the location of inital config for osohm container

### PR DESCRIPTION
**Pull request by ivanhorvath**

### CODE REVIEW

**Comment by joelddiaz** [comment]
LGTM :+1:

File: docker/oso-host-monitoring/centos7/root/config.yml [hunk 1]
```diff
@@ -20,6 +20,24 @@
...
+
+  - name: copy all yaml files found in /container_setup to /etc/openshift_tools
+    copy:
+      src: "{{ item }}"
+      dest: /etc/openshift_tools/
+      owner: root
+      group: root
+      mode: 0644
+    with_fileglob:
+        - /container_setup/*
```
Comment (twiest) [inline docker/oso-host-monitoring/centos7/root/config.yml]: I don't think this is what we were talking about, right?

Comment (twiest) [inline docker/oso-host-monitoring/centos7/root/config.yml]:
``` yaml

- include_vars: "{{ item }}"
  with_fileglob:
  - /container_setup/*.yml


- copy:
    content: "{{ etcd_metrics_config }}"
    dest: /etc/openshift_tools/etcd_metric.yml
  when: etcd_metrics_config is defined

```

**Comment by twiest** [comment]
:+1:

**Pull request merged**